man/man8/zpool.8

```diff
@@ -1926,8 +1926,10 @@ result in partially resilvered devices unless a second scrub is performed.
 .Ar pool Ar device Ns ...
 .Xc
 Removes the specified device from the pool.
-This command currently only supports removing hot spares, cache, log
-devices and mirrored top-level vdevs (mirror of leaf devices); but not raidz.
+This command supports removing hot spare, cache, log, and both mirrored and
+non-redundant primary top-level vdevs, this includes dedup and special vdevs.
```

Comment:
I think it would flow better with "this includes" -> "including"

Reply:
Sure, that flows a little better.